.github/autobuild/linux_deb.sh

```diff
@@ -50,10 +59,10 @@ setup_cross_compiler() {
     sudo update-alternatives --install "/usr/bin/${ABI_NAME}-g++" g++ "/usr/bin/${ABI_NAME}-g++-${GCC_VERSION}" 10
```

Comment:
Also... I take it line 50 in `setup_cross_compilation_apt_sources`
```
sudo sed -re 's/^deb /deb [arch=amd64,i386] /' -i /etc/apt/sources.list
```
doesn't need magic for arm, then?

Reply:
The magic for arm is in the line before the mentioned one.
That part is unchanged for this PR, btw.
I have now added a comment which explains the logic though.